backend/classes/Player.js

```diff
@@ -0,0 +1,20 @@
+class Player{
+    constructor(id){
+        this.id = id;
+
+        //what role he play in the game - Murderer, Doctor, Detective or null if not yet decided
+        this.role = null;
+
+        //what is he currently doing - playing, waiting, outside of the game etc..
+        this.activity = null;
+    }
+
+    static ROLES ={
+        YET_DECIDED: 0,
+        MURDERER: 1,
+        DOCTOR: 2,
+    }
+
+
+
+}
```

Comment:
newline at the end of the file is needed

Reply:
added, why?

Reply:
styling rules